### ISSUE TITLE
Add alt text to images

### ISSUE THREAD

**Issue by aleksisalonen** [City-of-Helsinki/linkedevents#350]
Under images, add a new text field:

"alt": "[alt text]",

**Comment by Doatti** [comment]
@aleksisalonen, what should be the maximum length of alt text?

**Comment by aleksisalonen** [comment]
Let's make it 320 characters.

**Comment by Doatti** [comment]
@aleksisalonen oops, I made the property name "alt_text" instead of "alt". Is this okay or should I change it to "alt"?

**Comment by aleksisalonen** [comment]
I suppose it doesn't really matter, as long as we use the same name for both video and image alt text. So no need to change it.
